src/middlewares/verifyToken.js

```diff
@@ -0,0 +1,80 @@
+import JWT from 'jsonwebtoken';
+import 'dotenv/config';
+import { errorResponse } from '../utils';
+import models from '../models';
+
+/**
+ * @class Authenticate
+ * @description authenticate tokens and roles
+ * @exports Authenticate
+ */
+class Authenticate {
+  /**
+   * Verify if token is valid
+   * @param  {object} req - The user request object
+   * @param  {object} res - The user res response object
+   * @param  {function} next - The next() Function
+   * @returns {String} req.userId - The user id
+   */
+  /* Uncomment this block to verify tokens
```

Comment:
this commented part of your _**authenticate code**_, (verifyToken) can another team mate use it for his/her task?

Reply:
Yes that's why I put it in a comment. It'll affect the test coverage since there is no route utilising it.